src/pages/nft/class/_classId/index.vue

```diff
@@ -511,6 +547,36 @@ export default {
       });
       logTrackerEvent(this, 'NFT', 'CopyShareURL(Details)', this.classId, 1);
     },
+    async handleClickFollow() {
```

Comment:
Can we refactor this follow action in wallet mixin, seems both NFT Class ID and NFT ID pages require it